templates.rst

```diff
@@ -1043,15 +1045,16 @@ the ``value`` is the Twig namespace, which is explained later:
     .. code-block:: php
 
         // config/packages/twig.php
-        $container->loadFromExtension('twig', [
+        use Symfony\Config\TwigConfig;
+
+        return static function (TwigConfig $twig) {
             // ...
-            'paths' => [
-                // directories are relative to the project root dir (but you
-                // can also use absolute directories)
-                'email/default/templates' => null,
-                'backend/templates' => null,
-            ],
-        ]);
+
+            // directories are relative to the project root dir (but you
+            // can also use absolute directories)
+            $twig->path('email/default/templates', null);
```

Comment:
Related to the singular <-> plural changes ... should this be instead the following?

```php
// directories are relative to the project root dir (but you
// can also use absolute directories)
$twig->paths([
    'email/default/templates' => null,
    'backend/templates' => null,
]);
```

Reply:
Since we do have a key here we treat it differently. 

```php
$twig->path('email/default/templates', null);
$twig->path('backend/templates', null);
// This will override the first one
$twig->path('email/default/templates', 'email');
```

```php
$twig->paths([
    'email/default/templates' => null,
    'backend/templates' => null,
]);

// It is not clear what would happen here, what will be overridden?
$twig->paths([
    'email/default/templates' => 'email',
]);
```

This may make more sense when the `$value` is not a scalar.

Reply:
Yes, now that I saw your example I agree with you and this is the best solution in this case. Thanks!